src/scripts/main.js

```diff
@@ -1,8 +1,37 @@
 'use strict';
 
+function gender(sex) {
+  if (sex === 'm') {
+    return 'Male';
+  }
+
+  return 'Female';
+};
+
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard tbody');
+let row = table.insertRow();
+
+for (let i = 0; i < people.length; i++) {
+  row = table.insertRow(i + 1);
+  table.appendChild(row);
+
+  row.insertCell(0)
+    .innerHTML = people[i].name;
+
+  row.insertCell(1)
+    .innerHTML = gender(people[i].sex);
+
+  row.insertCell(2)
+    .innerHTML = people[i].born;
+
+  row.insertCell(3)
+    .innerHTML = people[i].died;
+
+  row.insertCell(4)
+    .innerHTML = people[i].died - people[i].born;
 
-// write your code here
+  row.insertCell(5)
```

Comment:
anyway, it will be much clearer to use `insertAdjacentHTML` or `innerHTML` with `append`